src/pygambit/mixed.pxi

```diff
@@ -301,19 +301,13 @@ class MixedStrategyProfile:
         """Creates a copy of the mixed strategy profile."""
         return self._copy()
 
-    def all_zero_probs(self) -> bool:
-        """Returns True if at least one player has only zero probabilities."""
-        return self._all_zero_probs()
-
-    def negative_prob(self) -> bool:
-        """Returns True if at least one player has a negative \"probability\"."""
-        return self._negative_prob()
-
     def _all_zero_probs(self) -> bool:
```

Comment:
@tturocy I ended up with single implementations of _all_zero_probs and _negative_prob at the MixedStrategyProfile level. Do you see any reason for separate implementations in MixedStrategyProfileRational and MixedStrategyProfileDouble (maybe I missed something)?